incident_viewer_demo/src/App.scss

```diff
@@ -6,6 +6,26 @@ $background: #FFFFFF;
   text-align: center;
 }
 
+.showIncident {
+  margin-bottom: 20px;
```

Comment:
We technically haven't done this for our CSS, but usually these should be alphabetized. Up to you if you want to do this (but this is something that we should have and will be helpful once we add this to the client)